### ISSUE TITLE
Reduce mileage on scroll-wheel

### ISSUE THREAD

**Issue by originalsouth** [minvws/nl-kat-coordination#3654]
## User Story

In the current main (9aab66611b8924f40f0e648c0135cae7efbeeeb9), the [KAT-alogus](http://localhost:8000/en/0/kat-alogus/) scrolls all the way up every time a Boefje/Normalizer is toggled (enabled/disabled), to show the green "popup" message.

As a user I want to see the green "popup" message, without scrolling back to the place I was interacting with.

## Acceptance Criteria:

- [ ] Do not change position while toggling/modifing content
- [ ] Make sure the green "popup" message is visible to the user (user does not have to scroll in order to see it)

**Comment by underdarknl** [comment]
The ultimate solution would be to enable/disable the boefje via a Json api call and implement this as progressive enhancement.

**Comment by originalsouth** [comment]
> The ultimate solution would be to enable/disable the boefje via a Json api call and implement this as progressive enhancement.

Please not that this mechanism is also true for any other action on any another page that yields a green "popup" message.

**Comment by underdarknl** [comment]
Yup, im not a fan of the generic 'messages' solution that django provides, as it is not restful. Id much rather specifically handle the messages in the correct position (eg, near the action button/form), and with a restful pathway. Doing so in the post/get flow however requires that we signal the return page, which message to set in the url (via parameters), and where (via parameters).
When doing the actions via htmx, its a bit easier to also show the succes/error messages near the action.
